### DISCUSSION TITLE
I have success upload Wifipineapple Mark 7 to TP-Link C20 V5

### DISCUSSION THREAD

**Discussion by DRCRecoveryData** [xchwarze/wifi-pineapple-cloner#17] [Q&A]
Hi,

I upgrade rom from 8->16->32mb

![image](https://github.com/xchwarze/wifi-pineapple-cloner/assets/85211068/9c0e2c7d-c6e8-49a8-9cc7-92d88a936b73)

![image](https://github.com/xchwarze/wifi-pineapple-cloner/assets/85211068/b892498b-3f60-4caf-b65c-72da36e74ae0)

then i upload offical firmware of Mark 7:

https://downloads.hak5.org/pineapple/mk7

via breedweb recovery but got error:

![image](https://github.com/xchwarze/wifi-pineapple-cloner/assets/85211068/17ceca3f-f680-4602-8d07-6b1e8f81cc08)

i think this recovery firmware file is check push reset button then upload full firmware file

![image](https://github.com/xchwarze/wifi-pineapple-cloner/assets/85211068/72114ac5-0872-4dc2-8da9-7e9be08eecc5)

![image](https://github.com/xchwarze/wifi-pineapple-cloner/assets/85211068/6adfa56a-bf20-4b12-8f72-33d9641c17d8)

could you check how to bypass that recovery firmware check?

https://www.youtube.com/watch?v=LcAbDolN-v0

I found a detail of setup:

https://www.elecfans.com/d/2303636.html

**Comment by xchwarze** [comment]
Installing the software on a router is really not difficult, the problem is to pass the two or three hard checks that the platform has. Add to that the fact that it still has 3 more years of life, so launching something is to provoke them to add more checks and more complexes...

I already got this repo completely deleted once, I don't intend to do the same thing again, so I won't do anything with this platform.
However I am developing a private tool that can be used to do many of the things that pineapple does, it doesn't have a single line of code or any hak5 software. The test builds will be available only through Patreon.

**Reply by DRCRecoveryData** [reply]
when you release the test build on Patreon?

**Reply by xchwarze** [reply]
@DRCRecoveryData this month
